packages/api/src/routes/userFarmRoute.js

```diff
@@ -110,6 +110,13 @@ router.patch(
   userFarmController.updateWage(),
 );
 
+router.patch(
+  '/wage_do_not_ask_again/farm/:farm_id/user/:user_id',
+  hasFarmAccess({ params: 'farm_id' }),
+  checkScope(['edit:user_wage']),
```

Comment:
I don't know the answer to this one --- Is there better or different scope that maybe should be used like a "user_setting" or something that would be more appropriate since no user wage is being set?

Reply:
I don't think there's an option other than creating an appropriate one. What do you think about adding a permission for `edit:user_wage_do_not_ask_again`?

Reply:
I think an existing scope should be good. I am seeing the onboarding flags also in the userFarm table are set without permission in userFarmRoute -- which seems weird to me. Maybe one of the others used in userFarmRoute?

Reply:
We only have these scopes, and I can't reuse any of them here. The first one includes FW, which should not, the rest are dedicated to each column. I'll create a new one.
<img width="396" alt="Screenshot 2023-02-02 at 4 32 46 PM" src="https://user-images.githubusercontent.com/33141219/216482146-7a7b5134-eae5-4864-b4b9-205cc9b262fc.png">

Reply:
Yeah a FW would never see the prompt in the UI but really it is just a userFarm default setting which I think even a FW should be able to control on their own account? This is a weird one

Reply:
Okay, you must be right :) I'll go with edit:users